pkg/statedb/types.go

```diff
@@ -56,6 +56,19 @@ type Table[Obj any] interface {
 	// when anything in the table changes as more fine-grained notifications
 	// are not possible with a lower bound search.
 	LowerBound(ReadTxn, Query[Obj]) (iter Iterator[Obj], watch <-chan struct{})
+
+	// DeleteTracker creates a new delete tracker for the table
+	// starting from the given revision.
```

Comment:
is there a given revision?

Reply:
Leftover from when it actually did just take a revision. Reworded.